### PR TITLE
commit sklearn docker -  remove .tar

### PR DESCRIPTION
**Pull request by yguezpa**
## Status
Ready

## Related Issues
related: [Research] MTH Alert Prioritization #26297

## Description
This docker is basically built for sklearn package. It will be use to run automation for MTH incident prioritization model. 
It had only python with some basic ML packages: sklearn, pandas, numpy

### CODE REVIEW

File: docker/sklearn/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+FROM demisto/python3:3.7.5.5420
```
Comment (glicht) [inline docker/sklearn/Dockerfile:1]: Why not use our latest image based on python 3.8.3?

File: docker/sklearn/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+		make  rsync \
+        libpng-dev freetype-dev libexecinfo-dev libgomp lapack-dev \
+		libgcc libquadmath musl  \
+		libgfortran \
+        gcc \
+        python py-pip  \
+        gfortran \
+        musl-dev \
+  && apk add --no-cache libstdc++  \
+  && apk add --no-cache openblas-dev  \
```
Comment (glicht) [inline docker/sklearn/Dockerfile:16]: I think you can move this up to be part of the install which is for `.build-dependencies`

File: docker/sklearn/Dockerfile (hunk 1 above)
Comment (yguezpa) [reply docker/sklearn/Dockerfile:1]: The model is trained using python3.7. Using python 3.8 can cause come mistakes. @glicht

File: docker/sklearn/Dockerfile (hunk 2 above)
Comment (yguezpa) [reply docker/sklearn/Dockerfile:16]: I can't. If I remove those 2 packages the code won't work. 
libstdc++ is necessary for pandas 
openblas-dev for skikit learn 
@glicht

Comment (glicht) [reply docker/sklearn/Dockerfile:16]: I am not too familiar with this. But `openblas-dev` is a dev pacakge. For runtime you should probably be using `openblas`. I would try adding `openblas-dev` to the build-deependencies and installing separately `openblas`.

**Comment by yguezpa** [comment]
@glicht i removed the package openblas-dev
The size is now 165MB

File: docker/sklearn/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+COPY requirements.txt .
+
+RUN apk --update add --no-cache --virtual .build-dependencies python-dev build-base wget git g++ python3-dev\
+        openssh curl ca-certificates openssl less htop \
+		make  rsync \
+        libpng-dev freetype-dev libexecinfo-dev libgomp lapack-dev \
+		libgcc libquadmath musl  \
+		libgfortran \
+        gcc \
+        python py-pip  \
```
Comment (glicht) [inline docker/sklearn/Dockerfile:12]: I don't think this is needed

File: docker/sklearn/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,20 @@
+FROM demisto/python3:3.8.3.9324
+
+COPY requirements.txt .
+
+RUN apk --update add --no-cache --virtual .build-dependencies python-dev build-base wget git g++ python3-dev\
+        openssh curl ca-certificates openssl less htop \
+		make  rsync \
+        libpng-dev freetype-dev libexecinfo-dev libgomp lapack-dev \
```
Comment (glicht) [inline docker/sklearn/Dockerfile:8]: libgomp you are installing later on. So not needed here

**Comment by yguezpa** [comment]
@glicht i added the verify.py script. Should it contains something else ?

**Comment by glicht** [comment]
> @glicht i added the verify.py script. Should it contains something else ?

@yguezpa I would add: `sklearn.show_versions()`

**Pull request merged**